gothemis/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/cossacklabs/themis/gothemis
+
+go 1.12
```

Comment:
as I remember, this line adds the requirement of golang version and users with go_version != 1.12 will take errors.
we don't have any dependencies for specific go version or specific code. so I think we can drop this line, leave only `module github.com/....`

Reply:
`go mod init` inserted this line so I though it's okay to keep it. [The docs](https://golang.org/doc/go1.12#modules) say that it should be fine if the code actually compiles, except for Go versions 1.11~1.11.3 which will be broken.

I agree that we should drop this line until we have a minimum required version on Golang that we support.